bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/handler/TelenotBridgeHandler.java

```diff
@@ -100,6 +100,7 @@
     protected @Nullable ScheduledFuture<?> connectionCheckJob;
     protected @Nullable ScheduledFuture<?> refreshSendDataJob;
     protected @Nullable ScheduledFuture<?> connectRetryJob;
+    protected @Nullable ScheduledFuture<?> updateTelenotClockJob;
```

Comment:
You have to make sure you cancel that job ob thing disposal (like the other jobs).